frontends/qt/server/server.go

```diff
@@ -131,7 +132,7 @@ func serve(
 			DeviceInfosFunc:     usb.DeviceInfos,
 			SystemOpenFunc:      system.Open,
 			UsingMobileDataFunc: func() bool { return false },
-			NativeLocaleFunc:    func() string { return "" },
+			NativeLocaleFunc:    func() string { return C.GoString(preferredLocale) },
```

Comment:
This is a closure, so `preferredLocale` from C++ might already be popped of the stack by the time this function is called?

Reply:
In C++ the `serve` is actually called from main so it's never popped from the stack until exit. But you're right, I guess I got lucky! - we could potentially move the serve elsewhere in the future and this might suddenly start crashing.

Reply:
Ah true. I guess I was confused by:

`preferredLocale.toStdString().c_str()`

Which is kind of like a local var before the function call:

`auto cstr = preferredLocale.toStdString().c_str();`

which also lives until the end, but I kinda assumed that as a function argument, maybe it could be cleaned up earlier than when main() exits. On 2nd thought that sounds very wrong ;)

Reply:
Done